modules/analysis-common/build.gradle

```diff
@@ -37,3 +37,16 @@ artifacts {
 tasks.named("yamlRestCompatTestTransform").configure { task ->
   task.replaceValueInMatch("tokens.0.token", "absenț", "romanian")
 }
+
+tasks.named("yamlRestTest").configure {
+  if (buildParams.getRuntimeJavaVersion().map{ it.majorVersion.toInteger() }.get() >= 24 ||
+    "-Des.entitlements.enabled=true".equals(System.getProperty("tests.jvm.argline"))) {
+    systemProperty 'tests.rest.blacklist',
+      [
+        // this test relies on security manager, which doesn't exist in JDK 24.
+        // and entitlements don't yet replace the functionality.
+        // see https://github.com/elastic/elasticsearch/issues/119130
```

Comment:
Should we put "awaitsfix" in some for or another here in this comment so we can easily find this later?

Reply:
I don't know of an awaitsfix thing in gradle files, but I can certainly change the text to AWAITSFIX

Reply:
Yeah, that's all I meant. It just makes it easier to search for them.